core/trino-main/src/main/java/io/trino/operator/DriverFactory.java

```diff
@@ -32,7 +32,7 @@ public class DriverFactory
     private final int pipelineId;
     private final boolean inputDriver;
     private final boolean outputDriver;
-    private final List<OperatorFactory> operatorFactories;
+    private List<OperatorFactory> operatorFactories;
```

Comment:
Move below final fields. Make it `@GuardedBy("this")` as this class is thread safe

Reply:
I did that field `volatile`. If you annotate this field `@GuardedBy("this")` you need put all accesses under `synchronized` what makes code ugly, specifically for `io.trino.operator.DriverFactory#createDriver`. It is sufficient to make this field `volatile`.